qr-code-auth/README.md

```diff
@@ -0,0 +1,277 @@
+# Collection - QrCodeAuth
+
+## API Endpoints
+
+|                                       Route                                        |                                         Description                                          |
+| :--------------------------------------------------------------------------------: | :------------------------------------------------------------------------------------------: |
+|                      [POST /qr-code-auth](#post-qr-code-auth)                      |                            Creates a new authentication document.                            |
+|                       [GET /qr-code-auth](#get-qr-code-auth)                       |           Retrieves all the authentication document belonging to a specified user            |
+| [PATCH /qr-code-auth](#patch-qr-code-authauthorization_statusauthorization_status) | Updates the authorization_status field of an existing qr-code-auth document for the specified user. |
+
+## POST /qr-code-auth
+
+    Creates a new qr-code-auth document.
+
+- **Params**  
+  None
+- **Query**
+  None
+- **Body**
+  - Attributes
+    - **user_id** (required, string): Specifies the user id for the document.
+    - **device_info** (required, string): Specifies the device information associated with the authentication.
+- **Headers**
+  - Content-Type: application/json
+- **Cookie**
+  None
+- **Success Response:**
+  - **Code:** 201
+    - **Content:**
+      ```json
+      {
+        "message": "String",
+        "data": {
+          "user_id": "String",
+          "device_info": "String",
+          "authorization_status": "String",
+          "access_token": "String"
+        }
+      }
+      ```
+- **Error Response:**
+
+  - **Code:** 400
+
+    - **Content:**
+      ```json
+      {
+        "message": "Bad Request"
+      }
+      ```
+
+  - **Code:** 409
+
+    - **Content:**
+      ```json
+      {
+        "message": "The authentication document has already been created"
+      }
+      ```
+
+  - **Code:** 500
+    - **Content:**
+      ```json
+      {
+        "message": "The server has encountered an unexpected error. Please contact the administrator for more information."
+      }
+      ```
+
+- **Example for user authentication document creation request:**
+  POST /qr-code-auth<br/>
+  Content-Type: application/json<br/>
+  Request-Body:<br/>
+
+  ```json
+  {
+    "device_info": "t5k77PHnuDSrgEzvMJAj",
+    "user_id": "NLFSj7Kz30oHgolfIZtJ"
+  }
+  ```
+
+  Response :
+  Status 201<br/>
+  Content-Type: application/json<br/>
+
+  ```json
+  {
+    "data": {
+      "user_id": "SooJK37gzjIZfFNH0tlL",
+      "device_info": "t5k77PHnuDSrgEzvMJAj",
+      "authorization_status": "NOT_INIT",
```

Comment:
isn't the user `authorized`, because we are returning an `acesstoken`, why are we sending `NOT_INIT` instead? I might be wrong in understanding this

Reply:
`authorization_status` is a different than `acesstoken`. The `authorization_status` is the status that user will change themselves when they authorise themselves from the mobile app.